src/atopile/components.py

```diff
@@ -81,66 +55,111 @@ class NoMatchingComponent(errors.AtoError):
     title = "No component matches parameters"
 
 
-@cache
-def _get_generic_from_db(component_addr: str) -> dict:
+# Define the cache file path
+repo = Repo(".", search_parent_directories=True)
```

Comment:
I think the comment I left on Gitlab still applies here, it makes the CLI inconsistent. Basically, this means your CWD needs to be inside the project you want to work on and you can't just say `ato build path/to/project/file.ato...`

Reply:
Done